syft/artifact/relationship.go

```diff
@@ -1,20 +1,19 @@
-package pkg
+package artifact
 
 const (
 	// OwnershipByFileOverlapRelationship indicates that the parent package owns the child package made evident by the set of provided files
 	OwnershipByFileOverlapRelationship RelationshipType = "ownership-by-file-overlap"
 )
 
+type Identifiable interface {
+	Identity() ID
```

Comment:
nit / curious: Could we call the method `ID()`? This to me would read as a conventional Go "getter", where the method name matches the type of what's returned. The way I think of it, we're getting the "ID" (the identification), not the "identity", which would be more broad. What are your thoughts?

Reply:
I originally did have this as ID, one of the soft issues with this is that Package has a field called ID that needs to be exported. Up for naming suggestions here.

Reply:
Interesting. Is `Package.ID` a field that might be going away soon? (I'm vaguely remembering this from a conversation last week...)

Reply:
Possibility, but not finalized. We could make that change on this PR. I'll review tomorrow and get a more concrete answer.

Reply:
Okay cool! Just exploring. I'll think on a method name, too.

Reply:
`Identifier`? Always a good excuse to follow the `er` convention and an `Identifier` implementing `Identify` feels correct?

Reply:
I think I'm going to try and go the direction of an `ID()` method as a unravel the package ID assignments in the `pkg.Catalog` and start relying on the new fingerprint method.

Reply:
Identity has been renamed to ID 👍